pkgs/development/python-modules/ducc0/default.nix

```
@@ -2,7 +2,7 @@
 
 buildPythonPackage rec {
   pname = "ducc0";
-  version = "0.31.0";
+  version = "0.32.0";
 
   disabled = pythonOlder "3.7";
```

```suggestion
  disabled = pythonOlder "3.8";
```
Could you please update this?
https://gitlab.mpcdf.mpg.de/mtr/ducc/-/blob/ducc0_0_32_0/pyproject.toml?ref_type=tags#L2

---

Done, thanks for the review!